common/proc_allowing-external-access-knative-services-OCP-4x.adoc

```diff
@@ -0,0 +1,48 @@
+// This module is included in the following assemblies:
+//
+// assembly_knative-OCP-4x.adoc
+
+
+[id='allowing-external-access-knative-services-OCP-4x_{context}]
+= Allowing external access to Knative services on OCP 4.0
+
+Use a configured hostname to access Knative services on the OpenShift 4.0 cluster
+
+.Procedure
+
+. Updating Knative's config-domain ConfigMap and replacing "example.com" with your domain's suffix.
+
+NOTE: This will not be needed once the new operator capabilities are enabled to set the value automatically.
+
+.. Find out your cluster's domain suffix <YOUR CLUSTER SUFFIX> with the following command.
+
+   `oc get ingresses.config.openshift.io cluster -o jsonpath="{.spec.domain}"`
+   
+.. Edit Knative's config-domain ConfigMap and replace "example.com" with <YOUR CLUSTER SUFFIX>
+
+   `oc edit configmap config-domain -n knative-serving`
+
+[start=2]
+. Creating an OpenShift Route pointing to the istio-ingressgateway for each of your Knative Service. 
+
```

Comment:
Add a brief explanation about;
- why we need the route / what istio-ingressgateway is used for and why we need to point to it?
- does this need to be done just once or multiple times for different Knative Services?